lib/twitter/index.js

```diff
@@ -10,7 +10,7 @@ const getTweetById = async id => {
   }
 
   const response = await fetch(
-    `https://api.twitter.com/1.1/statuses/show.json?id=${id}`,
+    `https://api.twitter.com/1.1/statuses/show.json?id=${id}&tweet_mode=extended`,
```

Comment:
Not new but only noticing now: id should be encoded with `encodeURIComponent`.

But I'd use

```js
const {format} = require('url')

`https://api.twitter.com/1.1/statuses/show.json${format({
  query: {
    id,
    tweet_mode: 'extended'
  }
})}`
```